documents/Code of Conduct.md

```diff
@@ -0,0 +1,45 @@
+
+<div class="page-break"></div>
+
+1. **Scope and Application**
+   1. All Members of the WCA Community are automatically bound by and required to comply with all of the provisions of the Code of Conduct. Members of the WCA Community shall be deemed to have agreed:
+      1. that it is their personal responsibility to familiarize themselves with all of the requirements of the Code of Conduct, including what conduct constitutes an offense under the Code of Conduct;
+      2. to submit to the exclusive jurisdiction of any WCA Delegate, WCA Committee or WCA Board of Directors convened under the Disciplinary Procedure to hear and determine charges brought (and any appeals in relation thereto) pursuant to the Code of Conduct; and
+      3. not to bring any proceedings to any court or other forum that are inconsistent with the foregoing submission to the jurisdiction of WCA Delegates, WCA Committees or WCA Board.
+   2. While taking into account Article 1.1, the WCA and Members of the WCA Community shall be responsible for promoting Code of Conduct awareness and education amongst Members of the WCA Community; in particular WCA Officials.
+2. **Conduct**
+   1. Every member of the WCA Community must behave with respect towards the representatives of the WCA, other Registered Speedcubers, spectators, press and partners. This applies to behavior during WCA Competitions and also online, on official WCA platforms. Members of the WCA Community are expected to conduct themselves according to the following values:
+      1. Compassion: treat others as you would like to be treated.
+      2. Integrity: be honest and compete fairly.
+      3. Respect: show respect to all other humans.
+   2. Possible violations include, but are not limited to:
+      1. Hate speech, threats, offensive behavior, or verbal abuse related to sex, gender identity and expression, sexual orientation, race, ethnicity, disability, physical appearance, age, or religion.
+      2. Stalking or intimidation (physically or online).
+      3. Spamming, raiding, hijacking, or inciting disruption of streams or social media.
+      4. Posting, or threatening to post, other people’s personal information.
+      5. Sexual harassment. This includes unwelcome sexualized comments, jokes, and sexual advances.
+      6. Any action or behavior that can result in damage for the reputation of the WCA.
+      7. Advocating for, or encouraging, any of the above behavior.
+   3. The status of the WCA as a 100% volunteer-run nonprofit must be upheld by all Community Members. This means that:
+      1. Finances for WCA Competitions must be handled according to the WCA Code of Ethics, section “Finances”.
+      2. Leftover money must not be retained by Organizers who are not  Regional Organizations or WCA Delegates, who must abide by the Code of Ethics (2.3.1). Leftover money shall thus be reinvested and used for Community activities.
```

Comment:
Would it be possible that an organization (who is not a Regional Organization) to retain the leftover money?